lively.ide/studio/controls/body.cp.js

```diff
@@ -143,7 +140,7 @@ export class BodyControlModel extends PropertySectionModel {
   activate () {
     this.view.layout = this.view.layout.with({ padding: rect(0, 10, 0, 10) });
     this.view.master = this.activeSectionComponent; // eslint-disable-line no-use-before-define
-    this.addDynamicProperty();
+    this.addDynamicProperty(false, false);
```

Comment:
```suggestion
    this.addDynamicProperty(null, false);
```
I know that JS does not care (which is a feature by the way :joy: :upside_down_face:), but I like to use `false` for actual boolean values. Here, we want to explicitly pass *nothing*, for which I propose to use `null`. This way, it is also possible to differentiate between explicit and implicit setting a value to nothing (`null` vs `undefined`).